### PR TITLE
#5 sse being blocked by heroku

### LINKED ISSUE #5: SSE being blocked by Heroku

### PR DESCRIPTION
**Pull request by alexeymaslov**
* add reconnect logic if sse connection has errored (which always happens after 1 min of idling)
* split index.ts into separate functions a bit
* added gitattributes to enforce lf line endings on git checkout

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,11 @@
     "packages/*"
   ],
   "scripts": {
-    "deploy to heroku": "git push heroku master",
+    "deploy branch to heroku": "git push heroku 5-SSE_being_blocked_by_Heroku:master",
```
Comment (Axroy) [inline package.json]: do we need this in master branch?

Comment (alexeymaslov) [reply package.json]: I find it useful to have a specimen on how to deploy not master branch. Ideally branch name should be resolvable via smth like ${current_branch_name}. Do you know if this is possible with git?

Comment (Axroy) [reply package.json]: Maybe try this https://stackoverflow.com/questions/14031970/git-push-current-branch-shortcut ?

Comment (Axroy) [reply package.json]: I don't really know where are those scripts executed, though 🙃

Comment (alexeymaslov) [reply package.json]: Has changed to use 'HEAD' as alias for current branch name

**Pull request merged**